src/components/MessagesList/MessagesList.vue

```diff
@@ -238,14 +238,14 @@ export default {
 	},
 
 	watch: {
-		isWindowVisible(visible) {
-			if (visible) {
-				this.onWindowFocus()
-				// FIXME: the sidebar chat takes much longer to open, this is why we need a higher value here
-				// need to investigate why the sidebar takes that long to open and is not even animated
-				window.setTimeout(() => this.scrollToFocussedMessage(), 100)
-			}
-		},
+		// isWindowVisible(visible) {
+		// 	if (visible) {
+		// 		this.onWindowFocus()
+		// 		// FIXME: the sidebar chat takes much longer to open, this is why we need a higher value here
+		// 		// need to investigate why the sidebar takes that long to open and is not even animated
+		// 		window.setTimeout(() => this.scrollToFocussedMessage(), 100)
```

Comment:
This is the part we need to remove @nikola-gladovic

Reply:
Ok, sure